### ISSUE TITLE
Comparison between this and `ase.io.read`/`write`

### ISSUE THREAD

**Issue by Linux-cpp-lisp** [libAtoms/extxyz#9]
Hello,

Superficially, it seems like this should be a drop-in replacement for `ase.io.read`/`write` that just moves the parsing into C for speed (which is great!). Are there any pitfalls or other considerations, however, to be aware of when replacing `ase.io.read`/`write` with `libextxyz`?

Thanks!

**Comment by cortner** [comment]
Hi Alby - my understanding is that this is indeed how it is intended, but I think only @jameskermode can say more with confidence.

**Comment by jameskermode** [comment]
Yes, that’s right, although note there are some slight changes to the syntax. This implementation follows the specification set out in the radme, while the ase.io is not formalised. 

We intend to switch over ASE to use this library before too log.

**Comment by jameskermode** [comment]
As discussed: this library follows the formalised extxyz spec (README); ASE's reader predates it and we intend to migrate ASE onto this library. Closing as answered.
